templates/vas.conf.erb

```diff
@@ -38,7 +38,7 @@
 [vasypd]
  search-base = <%= @nismaps_ou %>
  split-groups = true
- update-interval = 1800
+ update-interval = <%= @vas_conf_vasypd_update_interval_real %>
```

Comment:
After $vas_conf_vasypd_update_interval has been converted to an real integer, please use $vas_conf_vasypd_update_interva_integer.